plugins/woocommerce-admin/client/task-lists/fills/shipping/index.js

```diff
@@ -236,7 +235,8 @@ export class Shipping extends Component {
 		};
 
 		const requiresJetpackConnection =
-			! isJetpackConnected && countryCode === 'US';
+			! isJetpackConnected &&
+			pluginsToActivate.includes( 'woocommerce-services' );
```

Comment:
Although there used to be a country code = US condition here, the only shipping extension that is recommended for US-based stores is WooCommerce Shipping & Tax so this is effectively the same for the country condition, while allowing us to be more specific about which plugin we're talking about (WCS&T vs. WCShip).